images/scripts/centos-8-stream.bootstrap

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
+
+BASE=$(dirname $0)
+
+# HACK using the url as a location is broken, fix this as soon as that works
```

Comment:
What does that mean? The URL looks okay to me, is that not going to be stable?

Reply:
The url is stable, but I'd prefer using the url as `--location` for virt-install rather than the iso image.